### ISSUE TITLE
0.10.0 broke assume-role

### ISSUE THREAD

**Issue by otterley** [segmentio/aws-okta#31]
```
SharedConfigAssumeRoleError: failed to load assume role for arn:aws:iam::xxxxx:role/RoleName, source profile has no shared credentials
```

0.9.0 works fine.

**Comment by dfuentes** [comment]
I'm not able to repro this on linux or mac.  Can you give some more info on how you installed (build from source, github release, brew release, etc)?

This is especially odd considering nothing changed w.r.t assume role (only change between 0.9 and 0.10 was handling more okta factor types).

**Comment by otterley** [comment]
I installed from the binary release uploaded to GitHub.  Running macOS 10.13.3 (High Sierra).

**Comment by otterley** [comment]
This problem only occurs with packer.

We discovered that unsetting the `AWS_PROFILE` environment variable addresses the issue.

**Comment by dfuentes** [comment]
This has been fixed in v0.11.0
